src/main/java/com/microsoft/graph/content/BatchResponseStep.java

```diff
@@ -61,7 +64,13 @@
         final GraphErrorResponse error = serializer.deserializeObject((JsonElement)body, GraphErrorResponse.class);
         if(error == null || error.error == null) {
             return serializer.deserializeObject((JsonElement)body, resultClass);
-        } else
-            throw GraphServiceException.createFromResponse("", "", new ArrayList<>(), "", headers, "", status, error, false);
+        } else {
```

Comment:
Is a unit test possible for this change?

Reply:
sure! there you go. :)